docs/docs/whatsnew.md

```diff
@@ -1,5 +1,19 @@
 # <b>Important updates</b>
 
+## v1.2.26
+
+### Support SBOM generation
+
+Software Bill of Materials (SBOM) is a list of components, libraries, and modules info that are required to build a software, and supply chain relationships between them. An SBOM also lists the licenses that govern those components, versions of the components used in the codebase, and their patch status.
+
+With SBOM, teams can quickly identify any associated security or license risks of codebase.
+
+Native builders `npm, maven and golang` and `mbt build, mbt sbom-gen` command are upgraded to support SBOM generation. For `java or nodejs` type module, or module's `build-parameters.builder` attribute value is `npm, maven or golang`, SBOM content will be generated and merged into one file. At present, only xml formate SBOM file is supported.
+
+Module configuration can be referenced in [configuration.md](https://github.com/SAP/cloud-mta-build-tool/blob/master/docs/docs/configuration.md) 
+
+SBOM generation command `mbt build` and `mbt sbom-gen` can be referenced in  [usage.md](https://github.com/SAP/cloud-mta-build-tool/blob/master/docs/docs/usage.md)
```

Comment:
The SBOM generation commands `mbt build` and `mbt sbom-gen` can be referenced in the  [usage.md](https://github.com/SAP/cloud-mta-build-tool/blob/master/docs/docs/usage.md) file.